tests/vectordb/test_zilliz_db.py

```diff
@@ -0,0 +1,191 @@
+# ruff: noqa: E501
+
+import os
+from dotenv import load_dotenv
+
+import unittest
+
+from embedchain import App
+from embedchain.config import ZillizDBConfig
+from embedchain.vectordb.zilliz import ZillizVectorDB
+
+load_dotenv()
+
+
+# to run tests, provide the URI and TOKEN in .env file
+class TestZillizVectorDBClient(unittest.TestCase):
+    # def test_init_with_uri_and_token(self):
+    #     """
+    #     Test if the `ZillizVectorDB` instance is initialized with the correct uri and token values.
+    #     """
+    #     uri = os.getenv("ZILLIZ_CLOUD_URI")
+    #     token = os.getenv("ZILLIZ_CLOUD_TOKEN")
+    #     config = ZillizDBConfig(uri=uri, token=token)
+
+    #     db = ZillizVectorDB(config=config)
+    #     db_config = db.config
+    #     self.assertEqual(db_config.uri, uri)
+    #     self.assertEqual(db_config.token, token)
+
+    def test_init_without_uri(self):
+        # Make sure it's not loaded from env
+        try:
+            del os.environ["ZILLIZ_CLOUD_URI"]
+        except KeyError:
+            pass
+        # Test if an exception is raised when ZILLIZ_CLOUD_URI is missing
+        with self.assertRaises(AttributeError):
+            ZillizVectorDB()
+
+    def test_init_without_token(self):
+        # Make sure it's not loaded from env
+        try:
+            del os.environ["ZILLIZ_CLOUD_TOKEN"]
+        except KeyError:
+            pass
+        # Test if an exception is raised when ZILLIZ_CLOUD_TOKEN is missing
+        with self.assertRaises(AttributeError):
+            ZillizVectorDB()
+
+    def test_init_invalid_cred(self):
+        # Test if an exception is raised when provided invalid credentials
+        with self.assertRaises(ValueError):
+            uri = "ululululu"
+            token = "random12345"
+            config = ZillizDBConfig(uri=uri, token=token)
+            ZillizVectorDB(config=config)
+
+
+# if using free tier, run only one test function at a time while commenting other test functions
+# if using free tier to test the below, then comment all the functions except the function you want to test
+# class TestZillizDBCollection(unittest.TestCase):
```

Comment:
Instead of commenting the tests, can you please mock the client and test accordingly?